.circleci/config.yml

```diff
@@ -78,8 +78,9 @@ jobs:
             export PATH="/tmp/go/bin:$PATH"
             export GOPATH=/tmp/go
             cd /tmp/go/src/github.com/FactomProject/factomd/
-            go test $(glide nv | grep -v Utilities | grep -v longTest | grep -v peerTest | grep -v simTest) -vet=off -v -cover -coverprofile=coverage.out
+            go test $(glide nv | grep -v Utilities | grep -v longTest | grep -v peerTest | grep -v simTest | grep '...') -vet=off -v -cover -coverprofile=coverage.out
```

Comment:
grep '...' matches any three characters. Probaly you want grep -E "[\.]{3}" which matches three . characters or ... '\.\.\.' though you may have to double \ that to make it work in a script.

Reply:
I fixed this in latest version - going to just export this single test rather than getting this branch rebased